File: backend/src/main/java/com/java/NBE4_5_1_7/domain/community/post/dto/EditPostRequestDto.java
package com.java.NBE4_5_1_7.domain.community.post.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EditPostRequestDto {
    private Long postId;
    private String title;
    private String content;
}

File: backend/src/main/java/com/java/NBE4_5_3_7/domain/community/post/dto/EditPostRequestDto.kt
package com.java.NBE4_5_3_7.domain.community.post.dto

class EditPostRequestDto {
    var postId: Long? = null
    var title: String? = null
    var content: String? = null

    constructor()

    constructor(postId: Long?, title: String?, content: String?) {
        this.postId = postId
        this.title = title
        this.content = content
    }
}
